superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -1610,11 +1610,7 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
                 ariaLabel={t('Log retention')}
                 placeholder={t('Log retention')}
                 onChange={onLogRetentionChange}
-                value={
-                  typeof currentAlert?.log_retention === 'number'
-                    ? currentAlert?.log_retention
-                    : ALERT_REPORTS_DEFAULT_RETENTION
-                }
```

Comment:
Let's double check if this can actually go

Reply:
The default log_retention value is specified as a number, and the dropdown options include only integers.

Additionally, there backend validation schema only allows integers. There shouldn't be a situation which calls for `typeof !== 'number'`